stdlib/LinearAlgebra/src/adjtrans.jl

```diff
@@ -251,6 +251,20 @@ parent(A::AdjOrTrans) = A.parent
 vec(v::TransposeAbsVec) = parent(v)
```

Comment:
Noticed in editing, this line looks wrong, shouldn't it restrict to `<:Number`? Example:
```julia
julia> v = [[1 2; 3 4]];

julia> vt = transpose([[1 2; 3 4]]);

julia> vt[1]
2×2 transpose(::Matrix{Int64}) with eltype Int64:
 1  3
 2  4

julia> vec(vt)[1]
2×2 Matrix{Int64}:
 1  2
 3  4
 ```

Reply:
Yes, seems like that almost noticed in 04f4884873c and f7de6d242f2 before, but not fully. Would you be able to submit a fix? Thanks!

Reply:
Done!